lib/linguist/generated.rb

```diff
@@ -590,6 +591,20 @@ def generated_gamemakerstudio?
       return lines[2].match(/\"modelName\"\:\s*\"GM/) ||
              lines[0] =~ /^\d\.\d\.\d.+\|\{/
     end
+        
+    # Internal: Is this a generated GIMP C image file?
+    #
+    # GIMP saves C sources with one of two comment forms:
+    # * `/* GIMP RGB C-Source image dump (<filename>.c) */` (C source export)
+    # * `/*  GIMP header image file format (RGB): <filename>.h  */` (Header export)
+    #
+    # Return true or false
+    def generated_gimp?
+      return false unless ['.c', '.h'].include? extname
+      return false unless lines.count > 0
+      return lines[0].match(/\/\* GIMP [a-zA-Z0-9- ]+ C\-Source image dump \(.+?\.c\) \*\//) ||
+             lines[0].match(/\/\*  GIMP header image file format \([a-zA-Z0-9- ]+\)\: .+?\.h  \*\//)
```

Comment:
@MCJack123 Heads up, the unescaped dashes in `[a-zA-Z0-9- ]` were interpreted as literal dashes, not character ranges. I fixed them in cc66523 to silence warnings emitted by Ruby at runtime; just an FYI for the future. 👍